src/components/RegisterForm/RegisterForm.module.scss

```
@@ -1,94 +1,319 @@
-.formContainer {
-	box-sizing: border-box;
-	background-color: #fff;
-	border-radius: 8px;
-	border: 1px solid #e5e8e5;
-	max-width: 460px;
-	margin: 0 auto;
-	padding: 40px;
+.formContainerOpened {
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  max-width: 600px;
+  width: 100%;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+  border-radius: 4px;
+  border: 1px solid #e5e8e5;
+  background: var(--BG-white, #fff);
+  padding: 40px;
 }
 
 .formTitle {
-	color: #2c2e2c;
-	margin: 0 0 20px 0;
-	font-size: 32px;
-	font-weight: 500;
-	line-height: 1.4;
+  color: var(--text-main, #2f302e);
+  font-family: Rubik;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 130%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
-.form {
+.formText {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: var(--text-main, #2f302e);
+  font-family: Rubik;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 150%;
 }
 
 .inputInscription {
-	color: #2c2e2c;
-	margin: 0;
-	font-size: 16px;
-	font-weight: 400;
-	line-height: 1.25;
+  color: #2c2e2c;
+  margin: 0;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.25;
 }
 
 .input {
-	box-sizing: border-box;
-	background-color: #fff;
-	border: 1px solid #e5e8e5;
-	border-radius: 12px;
-	color: #2c2e2c;
-	font-size: 14px;
-	font-weight: 400;
-	line-height: 1.42;
-	margin: 8px 0 12px;
-	padding: 12px;
-	width: 100%;
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #e5e8e5;
+  border-radius: 12px;
+  color: #2c2e2c;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 1.42;
+  margin: 8px 0 12px;
+  padding: 12px;
+  width: 100%;
+  margin-bottom: 12px;
 }
 
 .input:last-of-type {
-	margin-bottom: 0;
+  margin-bottom: 0;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 435px;
+  height: 52px;
 }
 
 .checkboxContainer {
-	display: flex;
-	margin: 20px 0;
-	padding: 10px 0 10px 0;
+  display: flex;
+  margin: 20px 0;
+  padding: 10px 0 10px 0;
+  min-height: 52px;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 20px;
 }
 
 .checkbox {
-	margin: 0 10px 0 10px;
-	cursor: pointer;
+  margin: 0 10px 0 10px;
+  cursor: pointer;
 }
 
 .inputLabel {
-	color: #2c2e2c;
-	margin: 0;
-	font-size: 16px;
-	font-weight: 400;
-	line-height: 1.25;
-	cursor: pointer;
+  cursor: pointer;
+  color: var(--text-main, #2f302e);
+  text-align: center;
+  font-family: Rubik;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 150%;
+}
+
+.forgetButton {
+  color: var(--Main, #53772f);
```

какие-то цвета начинаются с большой буквы, а какие-то с маленькой, а какие-то цвета без переменной. Надо сделать чтобы все цвета были в переменных и начинались с маленькой буквы.
Я уверен на 99%, что у вас нет требований, чтобы приложение работало, если кто-то удалит все переменные из css. Поэтому нужно удалить дефолтные значения там, где вы берете значения из переменных. В коде все должно быть максимально строго и минималистично. Шаг влево, шаг вправо - расстрел. Тогда все проблемы и противоречия в коде и в требованиях быстро выявить. Во-вторых, чем меньше написано, тем меньше шансов, что оттуда вылезет баг. Или какое-нибудь хитрое поведение, про которое никто не будет знать, нужно оно или нет.
```suggestion
  color: var(--main);
```